### PR TITLE
Fixed profile page refreshing issue

### PR DESCRIPTION
**Pull request by mistikk**

### CODE REVIEW

File: src/screens/profile/container/profileContainer.js [hunk 1]
```diff
@@ -349,6 +352,7 @@ class ProfileContainer extends Component {
       selectedQuickProfile,
       user,
       username,
+      forceLoadPost,
     } = this.state;
     const { isDarkTheme, isLoggedIn, currency, navigation } = this.props;
     const activePage = (navigation.state.params && navigation.state.params.activePage) || 0;
```
Comment (ue) [inline src/screens/profile/container/profileContainer.js]: use get from lodash

File: src/screens/profile/container/profileContainer.js [hunk 2]
```diff
@@ -84,15 +85,13 @@ class ProfileContainer extends Component {
...
 
-    if (currentUsername) {
-      this._loadProfile(currentUsername);
-    }
-
     if (
-      activeBottomTab !== nextProps.activeBottomTab &&
-      nextProps.activeBottomTab === 'ProfileTabbar'
+      (activeBottomTab !== nextProps.activeBottomTab &&
+        nextProps.activeBottomTab === ROUTES.TABBAR.PROFILE) ||
```
Comment (ue) [inline src/screens/profile/container/profileContainer.js]: use get from lodash

File: src/screens/profile/container/profileContainer.js [hunk 3]
```diff
@@ -75,7 +76,7 @@ class ProfileContainer extends Component {
   };
 
   componentWillReceiveProps(nextProps) {
-    const { navigation, currentAccount, activeBottomTab, isLoggedIn } = this.props;
+    const { activeBottomTab, currentAccount, isLoggedIn, navigation } = this.props;
     const currentUsername =
       currentAccount.name !== nextProps.currentAccount.name && nextProps.currentAccount.name;
```
Comment (ue) [inline src/screens/profile/container/profileContainer.js]: use get from lodash

File: src/components/posts/view/postsView.js [hunk 4]
```diff
@@ -47,11 +47,12 @@ class PostsView extends Component {
...
 
   componentWillReceiveProps(nextProps) {
-    const { currentAccountUsername } = this.props;
+    const { currentAccountUsername, changeForceLoadPostState } = this.props;
 
     if (
-      currentAccountUsername !== nextProps.currentAccountUsername &&
-      nextProps.currentAccountUsername
+      (currentAccountUsername !== nextProps.currentAccountUsername &&
+        nextProps.currentAccountUsername) ||
```
Comment (ue) [inline src/components/posts/view/postsView.js]: use get from lodash

File: src/components/posts/view/postsView.js [hunk 5]
```diff
@@ -47,11 +47,12 @@ class PostsView extends Component {
   }
 
   componentWillReceiveProps(nextProps) {
-    const { currentAccountUsername } = this.props;
+    const { currentAccountUsername, changeForceLoadPostState } = this.props;
```
Comment (ue) [inline src/components/posts/view/postsView.js]: func name is not valuable

File: src/screens/profile/container/profileContainer.js (hunk 2 above)
Comment (mistikk) [reply src/screens/profile/container/profileContainer.js]: it has different condition, `get` not suitable for use in here

File: src/screens/profile/container/profileContainer.js (hunk 3 above)
Comment (mistikk) [reply src/screens/profile/container/profileContainer.js]: it has different condition, `get` not suitable for use in here

File: src/components/posts/view/postsView.js (hunk 5 above)
Comment (mistikk) [reply src/components/posts/view/postsView.js]: ?

File: src/screens/profile/container/profileContainer.js (hunk 3 above)
Comment (ue) [reply src/screens/profile/container/profileContainer.js]: get(nextProps, 'currentAccount.name') @mistikk

File: src/components/posts/view/postsView.js (hunk 5 above)
Comment (ue) [reply src/components/posts/view/postsView.js]: `changeForceLoadPostState` what its changing what is working for this function. I dont understand from name

**Comment by ue** [comment]
@mistikk has conflits

File: src/components/posts/view/postsView.js (hunk 5 above)
Comment (mistikk) [reply src/components/posts/view/postsView.js]: it is changing to ForceLoadPost state

**Comment by mistikk** [comment]
@ue updated

**Pull request merged**